teamcity/main.tf

```diff
@@ -0,0 +1,51 @@
+provider "aws" {
+  region = "%env.AWS_REGION%"
```

Comment:
Here and throughout this file, I think, we will have to use the format in which terraform can recognize environment variables. 

They are referenced as `"${var.ENV_VARNAME}"`

https://stackoverflow.com/a/36672931/18852